apps/snitch_core/lib/core/data/schema/variation_theme.ex

```diff
@@ -52,14 +52,18 @@ defmodule Snitch.Data.Schema.VariationTheme do
     |> put_assoc_option_types(params["option_type_ids"])
   end
 
-  defp put_assoc_option_types(changeset, option_type) when option_type in [nil, ""] do
```

Comment:
When creating/editing theme we can get `nil` and `""`. Both values means option types were not selected.
We can handle in this function clause, just return the changeset with errors and remove the code that saves empty list.